### PR TITLE
auto startRound 

### PR DESCRIPTION
**Pull request by shine2lay**
I also had to refactor some part of startRound to make it smooth

also currentRound starts at 1

addMember can only be called during round 1
@morelazers

### CODE REVIEW

File: .idea/dictionaries/shine.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
```
Comment (morelazers) [inline .idea/dictionaries/shine.xml]: Can you remove this file pls?

Comment (shine2lay) [reply .idea/dictionaries/shine.xml]: how this get in there?

Comment (shine2lay) [reply .idea/dictionaries/shine.xml]: i added .idea to git ignore does it help?

Comment (morelazers) [reply .idea/dictionaries/shine.xml]: Try `.idea/`

**Review by morelazers — APPROVED** [review]

**Pull request merged**